images/tier1/change-file.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
This seems like a mirror of the existing "topic-last" icon. Not good to mix up these paradigms...